src/features/projects/ProjectRegistForm.tsx

```diff
@@ -0,0 +1,46 @@
+import { createProject } from 'api';
+import React, { useCallback, useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+// DocumentList represents the list of documents in the application.
+export function ProjectRegistForm() {
+  const navigate = useNavigate();
+  const nameRef = useRef<HTMLInputElement | null>(null);
+  const handleSumbit = useCallback(
+    async (e) => {
+      e.preventDefault();
+
+      const newProject = await createProject(nameRef.current?.value!);
```

Comment:
Should we handle an exception for an empty name here?

Reply:
I checked it using the `required` attribute on `<input>`. Users must specify a value before the form can be submitted.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required

Reply:
Thank you for your answer. It seems to be checked only in the Server.